src/qml/DashBoard.qml

```diff
@@ -319,18 +320,25 @@ Drawer {
               id: temporalButton
               Layout.alignment: Qt.AlignVCenter
               visible: flatLayerTree.isTemporal
-              iconSource: mapSettings.isTemporal ? Theme.getThemeVectorIcon( 'ic_temporal_green_24dp' ) : Theme.getThemeVectorIcon( 'ic_temporal_black_24dp' )
-              bgcolor: "white"
+              iconSource: Theme.getThemeVectorIcon( 'ic_temporal_black_24dp' )
+              iconColor: mapSettings.isTemporal ? Theme.mainColor : Theme.mainTextColor
```

Comment:
what is `Theme.mainColor`, color of what? At least for me, it is confusing especially with `Theme.mainTextColor` as alternative in the condition. Probably `Theme.accentColor`, so at least the "main" word is different?

Reply:
Theme.mainColor is our beautiful lime green, the color to rule them all, the Main Color.

Reply:
I guess if I was more involved in the codebase I would have remembered the nickname of the Color of the Colors.